cli.py

```diff
@@ -0,0 +1,108 @@
+import click
+
+import os
+import json
+import asyncio
+from time import sleep
+
+from faust import cli
+
+from thoth.messaging.hash_mismatch import HashMismatchMessage
+from thoth.messaging.message_base import MessageBase
+from thoth.messaging.message_factory import message_factory
+from thoth.messaging.missing_package import MissingPackageMessage
+from thoth.messaging.missing_version import MissingVersionMessage
+from thoth.messaging.package_releases import PackageReleaseMessage
+from thoth.messaging.unresolved_package import UnresolvedPackageMessage
+
+ALL_MESSAGES = [
```

Comment:
What about moving this to https://github.com/thoth-station/messaging/blob/master/thoth/messaging/__init__.py? So we import messages once and you can import only ALL_MESSAGES from there. We can also avoid forgetting to import both places.